AllEvents.TicketManagement/src/Infrastructure/AllEvents.TicketManagement.Persistance/Repositories/EventRepository.cs

```diff
@@ -26,5 +27,10 @@ public async Task<List<Event>> GetPagedEventsAsync(int page, int pageSize)
                 .Take(pageSize)
                 .ToListAsync();
         }
+
+        public async Task<Event> GetByIdAsync(Guid id)
+        {
```

Comment:
@KKDinev20 fix the github action warning 
Nullability of reference types in return type of 'Task<Event> EventRepository.GetByIdAsync(Guid id)' doesn't match implicitly implemented member 'Task<Event?> IEventRepository.GetByIdAsync(Guid eventId)'